optional_plugins/golang/tests/test_nrunner_interface.py

```diff
@@ -0,0 +1,30 @@
+import json
+import sys
+
+from avocado import Test, fail_on
+from avocado.utils import process
+
+
+class Interface(Test):
```

Comment:
Maybe I'm missing something, but why this is being tested here? What part of this test is stressing the golang plugin?